Makefile

```diff
@@ -1,7 +1,8 @@
-RELEASE_DIR = /build/release
+BUILDDIR:=$(PWD)
```

Comment:
```suggestion
ifeq ($(BUILDDIR),)
BUILDDIR := $(shell pwd)
endif
```

Reply:
The `ifeq` seems unnecessary, just shelling out is enough.